vello/src/render.rs

```diff
@@ -194,61 +194,35 @@ impl Render {
         );
         let ptcl_buf =
             ResourceProxy::new_buf(buffer_sizes.ptcl.size_in_bytes().into(), "vello.ptcl_buf");
-        let reduced_buf = ResourceProxy::new_buf(
-            buffer_sizes.path_reduced.size_in_bytes().into(),
-            "vello.reduced_buf",
-        );
-        // TODO: really only need pathtag_wgs - 1
-        recording.dispatch(
-            shaders.pathtag_reduce,
-            wg_counts.path_reduce,
-            [config_buf, scene_buf, reduced_buf],
-        );
-        let mut pathtag_parent = reduced_buf;
-        let mut large_pathtag_bufs = None;
-        let use_large_path_scan = wg_counts.use_large_path_scan && !shaders.pathtag_is_cpu;
-        if use_large_path_scan {
-            let reduced2_buf = ResourceProxy::new_buf(
-                buffer_sizes.path_reduced2.size_in_bytes().into(),
-                "vello.reduced2_buf",
-            );
-            recording.dispatch(
-                shaders.pathtag_reduce2,
-                wg_counts.path_reduce2,
-                [reduced_buf, reduced2_buf],
-            );
-            let reduced_scan_buf = ResourceProxy::new_buf(
-                buffer_sizes.path_reduced_scan.size_in_bytes().into(),
-                "reduced_scan_buf",
-            );
-            recording.dispatch(
-                shaders.pathtag_scan1,
-                wg_counts.path_scan1,
-                [reduced_buf, reduced2_buf, reduced_scan_buf],
-            );
-            pathtag_parent = reduced_scan_buf;
-            large_pathtag_bufs = Some((reduced2_buf, reduced_scan_buf));
-        }
-
         let tagmonoid_buf = ResourceProxy::new_buf(
             buffer_sizes.path_monoids.size_in_bytes().into(),
             "vello.tagmonoid_buf",
         );
-        let pathtag_scan = if use_large_path_scan {
-            shaders.pathtag_scan_large
-        } else {
-            shaders.pathtag_scan
-        };
+        let reduced_buf = BufferProxy::new(
+            buffer_sizes.path_reduced.size_in_bytes().into(),
+            "reduced_buf",
```

Comment:
This and the one below should be prefixed with "vello." so that it is clear that they're from Vello when using the labels in various GPU tools.